include/deal.II/particles/particle_handler.h

```diff
@@ -668,7 +689,7 @@ namespace Particles
      * n_locally_owned_particle() indices.
      */
     IndexSet
-    locally_relevant_ids() const;
+    locally_owned_particles() const;
```

Comment:
I think that the function should be name to make it clear that it doesn't return *particles* but *particle ids*:
```suggestion
    locally_owned_particle_ids() const;
```